docker-compose.yml

```
@@ -1,121 +1,128 @@
-version: '3.8'
+version: "3.8"
 
 services:
+  parcellaire-importer:
+    container_name: parcellaire-express-importer
+    build:
+      context: ./importer
+    environment:
+      - POSTGRES_USER
+      - POSTGRES_PASSWORD
+      - POSTGRES_DB
+      - POSTGRES_HOST
+      - POSTGRES_PORT
+      - POSTGRES_SCHEMA
+      - DOWNLOAD_URL
+      - TEST_IMPORTER
+      - MAX_PARALLEL_DL
+    volumes:
+      - importer-data:/tmp
 
-    parcellaire-importer:
-        container_name: parcellaire-express-importer
-        build:
-            context: ./importer
-        environment:
-            - POSTGRES_USER
-            - POSTGRES_PASSWORD
-            - POSTGRES_DB
-            - POSTGRES_HOST
-            - POSTGRES_SCHEMA
-            - DOWNLOAD_URL
-            - TEST_IMPORTER
-            - MAX_PARALLEL_DL
-        volumes:
-            - importer-data:/tmp
+  parcellaire-postgis:
+    container_name: parcellaire-express-postgis
+    build:
+      context: ./postgis
+      args:
+        - POSTGRES_VERSION
+        - POSTGIS_VERSION
+    environment:
+      - POSTGRES_DB
+      - POSTGRES_PORT
+      - POSTGRES_USER
+      - POSTGRES_PASSWORD
+      # Exposition du port de la base de données pour test si nécessaire
+    ports:
+      - "${POSTGRES_PORT}:${POSTGRES_PORT}"
```

```suggestion
    # ports:
    #   - "${POSTGRES_PORT}:${POSTGRES_PORT}"
```
Only for testing purposes. Not needed for normal API use.